tools/firtool/firtool.cpp

```diff
@@ -135,11 +135,17 @@ static cl::list<std::string>
     passPlugins("load-pass-plugin", cl::desc("Load passes from plugin library"),
                 cl::CommaSeparated, cl::cat(mainCategory));
 
-static cl::opt<std::string>
-    highFIRRTLPassPlugin("high-firrtl-pass-plugin",
-                         cl::desc("Insert passes after parsing FIRRTL. Specify "
-                                  "passes with MLIR textual format."),
-                         cl::init(""), cl::cat(mainCategory));
+static cl::opt<std::string> initialFIRRTLPassPlugin(
+    "initial-firrtl-pass-plugin",
+    cl::desc("Insert passes just after parsing FIRRTL. Specify "
+             "passes with MLIR textual format."),
```

Comment:
Maybe clarify this (and other instances) this is meant to be the MLIR textual pipeline format, not MLIR text itself (representing the IR).